mkosi/__init__.py

```diff
@@ -4478,11 +4479,16 @@ def generate_key_cert_pair(args: Args) -> None:
     keylength = 2048
     expiration_date = datetime.date.today() + datetime.timedelta(int(args.genkey_valid_days))
 
-    for f in ("mkosi.key", "mkosi.crt"):
-        if Path(f).exists() and not args.force:
+    configdir = get_configdir(args)
+
+    for f in (configdir / "mkosi.key", configdir / "mkosi.crt"):
+        if f.exists() and not args.force:
             die(
                 f"{f} already exists",
-                hint="To generate new keys, first remove mkosi.key and mkosi.crt",
+                hint=(
+                    "To generate new keys, first remove "
+                    f"{configdir / 'mkosi.key'} and {configdir / 'mkosi.crt'}"
+                ),
```

Comment:
Let's not split this apart
```suggestion
                hint=(
                    f"To generate new keys, first remove {configdir / 'mkosi.key'} and {configdir / 'mkosi.crt'}"  # noqa: E501
                ),
```
or just keep it without configdir, since that can already be read of from the line above and having it twice there, this could become quite long on screen.